workorders.lic

```diff
@@ -28,7 +28,8 @@ class WorkOrders
     settings = get_settings
     crafting_data = get_data('crafting')
     @bag = settings.crafting_container('backpack')
-    info = crafting_data[discipline]
+    @bank_info = get_data('town')[settings.hometown]['deposit']['id']
+    info = crafting_data[discipline][settings.hometown]
```

Comment:
No, there would have been a problem and as I was looking at the easiest way
to correct this I decided on another approach. I'm looking at all the
crafting scripts that have room #'s and such hard coded in and fixing them
so they pull that info from the base-crafting yaml.

I'll make sure all of the scripts work from Crossing and then it'll be as
simple as updating the yaml for other locations. Please reject that pull
request.

On Mon, Sep 5, 2016 at 8:03 AM, Robert Herbig <notifications@github.com>
wrote:

> In workorders.lic
> <https://github.com/rpherbig/dr-scripts/pull/740#discussion_r77511987>:
>
> > @@ -28,7 +28,8 @@ class WorkOrders
> >      settings = get_settings
> >      crafting_data = get_data('crafting')
> >      @bag = settings.crafting_container('backpack')
> > -    info = crafting_data[discipline]
> > +    @bank_info = get_data('town')[settings.hometown]['deposit']['id']
> > +    info = crafting_data[discipline][settings.hometown]
>
> Will this work for Forging/Tailoring/Carving (i.e. the disciplines that
> aren't yet by-town)?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rpherbig/dr-scripts/pull/740/files/e15e421f61d655b5fb6daf056d53b577b3d024a2#r77511987>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARQKjoHyZkTnk92nhWT4iONy6zZncv_vks5qnAUkgaJpZM4J0zPF>
> .
>